modules/ROOT/pages/data-bridge.adoc

```diff
@@ -8,133 +8,122 @@ The data bridge is the secure connection from Anypoint API Community Manager to
+. Set *Issuer* to the client ID of the connected app you saved previously.
+. Set *Per User Subject* to the following formula.
 +
-This secret was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
+Replace `IDP_ISSUER` with the issuer from your IdP configuration. Replace `IDP_AUDIENCE` with the audience from your IdP configuration.
 +
-. Set *Authorize Endpoint URL* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/authorize+` .
-. Set *Token Endpoint URL* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+` .
-. Set *Default Scopes* to `full offline_access`.
+`"{IDP_ISSUER}:{IDP_AUDIENCE}|" &  $User.Username & "|" & $User.FirstName & "|" & $User.LastName & "|" & $User.Email & "|[\"Community User\"]"`
+. Set *Audiences* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
+. Set *JWT Signing Certificate* to the certificate you downloaded when you created a key pair in the previous steps.
+. In the *Callout Options* section, enable *Generate Authorization Header*.
+
+== Modify External Data Source
+
+. Click *Setup* > *External Data Source* > *Exchange*.
+. Set *URL* to `callout:acm_pkg__anypoint_mulesoft/odatabridge/odata.svc/`.
+. Go to the *Authentication* section.
+. Set *Identity* to *Anonymous*.
+. Set *Authentication Protocol* to *No Authentication Protocol*.
+. In the section *Custom HTTP Headers*, for the *Header Field Name* *X-FirstName*, ensure *Active* is disabled.
 . Click *Save*.
 
-NOTE: Remember to associate the external data source with this authentication provider.
+== User Configuration
 
-== Authenticate the Data Bridge [[authenticate-data-bridge]]
+Give your user the *Exchange Contributor* role:
 
-. Search for *External Data Source* in the *Quick Find* box and then click *External Data Sources*.
-. Locate *Exchange* and click *Edit*.
-. Update *Identity Type* to *Named Principal*.
-. Update *Authentication Protocol* to *OAuth 2.0*.
-. Update *Authentication Provider* to *Anypoint*.
-. Click *Save*.
-. If you don't have an active session in your Anypoint Platform organization, log in.
-. Verify the Exchange *External Data Sources* *Authentication Status* is *Authenticated*.
-. Search for *Named Credentials* in the *Quick Find* box and then click *Named Credentials*.
-. Locate *anypoint.mulesoft* and click *Edit*.
-. Update *Identity Type* to *Named Principal*.
-. Update *Authentication Protocol* to *OAuth 2.0*.
-. For *Authentication Provider*, select your newly created *Anypoint* authentication provider.
-. Enable *Start Authentication Flow on Save*.
-. Click *Save*.
+. In your Anypoint Platform organization, click *Access Management* > *Users*.
+. Select your user.
+. Click *Role* > *Add role by name*.
+. Add the role *Exchange Contributor*.
+
+Map community user to Anypoint role:
```

Comment:
Why?

Reply:
I think if a task topic has more than one part, they should be intro'd in the first sentence so that the audience knows it ahead of time.
You typically do not want more than one set of steps per task.